### ISSUE TITLE
startInstancePubSub 

### ISSUE THREAD

**Issue by zhukovnv84** [GoogleCloudPlatform/nodejs-docs-samples#1571]
use this instruction for create scheduled instance, using gcloud

https://cloud.google.com/scheduler/docs/start-and-stop-compute-engine-instances-on-a-schedule

After created functions - instance is not started. 
error in cloud fucntion  startInstancePubSub  - 

TypeError: First argument must be a string, Buffer, ArrayBuffer, Array, or array-like object. |  
-- | --

**Comment by jackson-lucas** [comment]
I was having the same problem, what I noticed is the cloud function code on the link is different from the code on Github.

The code on link make it seems there is `event.data.data`, which is false. To fix I used the code from the link but without `pubsubMessage` constant and passed `event.data` directly to `Buffer.from()` like the code on github (https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/functions/scheduleinstance/index.js#L36).

Hope it helps :)
